behave/features/cla_public/public_end_to_end.feature

```diff
@@ -0,0 +1,34 @@
+@cla_public
+Feature:
+  - Applying for legal aid on the "Check if I can get legal aid" website
```

Comment:
Just example what this files tests are doing, so something along the lines of GDS exit page guidance.